### PR TITLE
Add basic function calling example notebook.

### PR DESCRIPTION
**Pull request by MarkDaoust**
## Description of the change

Add a notebook with an example of basic function calling.

### CODE REVIEW

File: site/en/tutorials/function_calling_python_quickstart.ipynb [hunk 1]
```diff
@@ -0,0 +1,721 @@
+{
```
Comment (anirudh161) [inline site/en/tutorials/function_calling_python_quickstart.ipynb:1]: <p>Change the section title to Summary and merge the bullets into a paragraph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

**Review by anirudh161 — DISMISSED** [review]

File: site/en/tutorials/function_calling_python_quickstart.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply site/en/tutorials/function_calling_python_quickstart.ipynb:1]: Done.

File: site/en/tutorials/function_calling_python_quickstart.ipynb [hunk 2]
```diff
@@ -0,0 +1,721 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lEXQ3OwKIa-O"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://developers.generativeai.google/tutorials/function_calling_python_quickstart\"><img src=\"https://developers.generativeai.google/static/site-assets/images/docs/notebook-site-button.png\" height=\"32\" width=\"32\" />View on Generative AI</a>\n",
```
[the remark this answers is not in the document]
Comment (MarkDaoust) [reply site/en/tutorials/function_calling_python_quickstart.ipynb]: Done.

File: site/en/tutorials/function_calling_python_quickstart.ipynb [hunk 3]
```diff
@@ -0,0 +1,719 @@
+{
```
Comment (pcoet) [inline site/en/tutorials/function_calling_python_quickstart.ipynb:1]: <p>"Gamini" -&gt; "Gemini"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (pcoet) [inline site/en/tutorials/function_calling_python_quickstart.ipynb:1]: <p>"Setup" -&gt; "Set up"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (pcoet) [inline site/en/tutorials/function_calling_python_quickstart.ipynb:1]: <p>"funish" -&gt; "furnish"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

**Comment by pcoet** [comment]
Cool tutorial!

File: site/en/tutorials/function_calling_python_quickstart.ipynb (hunk 3 above)
Comment (MarkDaoust) [reply site/en/tutorials/function_calling_python_quickstart.ipynb:1]: done.

Comment (MarkDaoust) [reply site/en/tutorials/function_calling_python_quickstart.ipynb:1]: Done.

Comment (MarkDaoust) [reply site/en/tutorials/function_calling_python_quickstart.ipynb:1]: Finish. Done.

**Review by pcoet — APPROVED** [review]

**Pull request merged**